js/animation/presets.d.ts

```diff
@@ -45,7 +46,7 @@ export default class animationPresets {
      * @prevFileNamespace DevExpress.animation
      * @public
      */
-    getPreset(name: string): any;
+    getPreset(name: string): { animation?: animationConfig, device?: Device };
```

Comment:
I think we can move it to the separate interface `animationPreset` and document it